Tests/JsonApiSerializerTests.cs

```diff
@@ -162,6 +162,27 @@ public void AppliesSorting()
             Assert.Equal(expected, props);
         }
 
+        [Fact(DisplayName = "Applies including when includes specified")]
+        public void AppliesIncluding()
```

Comment:
It could be good to have a test scenario like this:

1. two resources that both have a relationship to the same third resource
2. both include it
3. it's only in the included hash once